app/controllers/api/databases.php

```diff
@@ -1221,6 +1200,8 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     ->label('event', 'databases.[databaseId].collections.[collectionId].attributes.[attributeId].create')
     ->label('scope', 'collections.write')
     ->label('audits.resource', 'database/{request.databaseId}/collection/{request.collectionId}')
+    ->label('usage.metric', 'collections.{scope}.requests.update')
+    ->label('usage.params', ['databaseId' => 'request.databaseId'])
```

Comment:
Can we be consistent with Shimon's syntax?
`{response.userId}`

Do we use value for just getting the plain value. Is `value.email` translated to just `email`? In that case we can just set `email` without the `{}`

Reply:
If i add `{}` it will just make it complex, i'll have to use regex to parse these, keeping it a plain array makes it more easy to use? Let me know what you think

Reply:
I think we should be consistent, pretty sure Shimon is already taken care of the parsing. Worst case, please assign a task to review the labels format later and come up with something consistent.